docviewer/src/render_doc.ts

```diff
@@ -51,6 +51,13 @@ export default async function renderDoc(
 			text = await docFetch(repo + '/' + version + '/' + file);
 		}
 
+		if (file.indexOf('widget-core') > -1) {
```

Comment:
Any reason to limit this to widget-core?

Reply:
Actually no, just I was testing it against that. Thanks!

Reply:
Updated